.travis.yml

```diff
@@ -39,7 +39,7 @@ script:
     set -e
     if [ "$TRAVIS_BRANCH" == "gh-pages" ]
     then
-      htmlproofer --http-status-ignore 405,999 --url-ignore "/.*localhost.*/","/.*vimeo\.com.*/" --file-ignore "/.*\/files\/.*/"
+      htmlproofer --assume-extension --http-status-ignore 405,999 --url-ignore "/.*localhost.*/","/.*vimeo\.com.*/","/.*gitter\.im.*/" --file-ignore "/.*\/files\/.*/"
```

Comment:
why do you need this `--assume-extension` option?

Reply:
it is so when you have links like `/hall-of-fame`, htmlproofer will assume the `.html` and find the `hall-of-fame.html` file (otherwise it will give errors about these links)